ops/scripts/deployer.sh

```diff
@@ -19,6 +19,17 @@ curl \
 yarn run deploy
 
 if [ -n "$DTL_REGISTRY_URL" ] ; then
+
+    # there's a resource race - where DTL can't start without an address manager address
+    # and deployer can't upload it's addresses without DTL being up
+    # so this little python server pushes the address manager into dtl.sh startup sequence
+    # so that DTL can initialize itself (and the diff between upstream is thus reduced to minimum)
+    # after this sequence the deployer exits gracefully
+    echo "Starting server in background."
+    python3 -m http.server \
+        --bind "0.0.0.0" 8082 \
+        --directory ./dist/dumps &
```

Comment:
the reason why this was added is because the initialization of `L1DataTransportService` in DTL goes through data validation of property `addressManager` - which in our version of the code was not set (because it was later pushed *into* DTL).

Pushing into DTL is preserved (DTL is still serving all data as before!) but the address manager address now gets pulled from the deployer to DTL in `dtl.sh` entrypoint.

Reply:
Very good!